PortFxCop/src/Tools/FileIssues/GitHub.cs

```diff
@@ -6,11 +7,27 @@ internal static class GitHub
     {
         private const string ApplicationName = "RoslynAnalyzerIssueFiler";
 
+        private static GitHubClient _gitHubClient;
+
+        private static GitHubClient GetGitHubClient(string token)
+        {
+            if (_gitHubClient == null)
+            {
+                _gitHubClient = new GitHubClient(new ProductHeaderValue(ApplicationName));
+                _gitHubClient.Connection.Credentials = new Credentials(token);
+            }
+
+            return _gitHubClient;
+        }
+
         internal static IIssuesClient GetIssuesClient(string token)
         {
-            var gitHub = new GitHubClient(new ProductHeaderValue(ApplicationName));
-            gitHub.Connection.Credentials = new Credentials(token);
-            return gitHub.Issue;
+            return GetGitHubClient(token).Issue;
+        }
+
+        internal static IIssuesLabelsClient GetIssuesLabelsClient(string token)
+        {
```

Comment:
There is a separate API for adding and removing individual labels from an issue.